tests/test_selector.py

```diff
@@ -213,7 +214,23 @@ def test_invalid_xpath(self):
         try:
             x.xpath(xpath)
         except ValueError as e:
-            assert xpath in str(e), "Exception message does not contain invalid xpath"
+            assert xpath in str(e),\
+                "Exception message does not contain invalid xpath"
```

Comment:
self.assertIn will show both values in the error message. I think it is more useful than this custom exception message.